os/fs/smartfs/smartfs.h

```diff
@@ -175,6 +175,10 @@
 #define INODE_STATE_FILE          (CONFIG_NXFFS_ERASEDSTATE ^ 0x22)
 #define INODE_STATE_DELETED       (CONFIG_NXFFS_ERASEDSTATE ^ 0xaa)
 
+/* Smartfs worbuffer maxuimum length */
```

Comment:
could you check spelling?
worbuffer -> workbuffer
maxuimum -> maximum

Reply:
Noted, thankyou.
Pending though for upcoming PRs.